MyMentorProject/src/main/java/com/Deny/MyMentorProject/Controllers/UserMainController.java

```diff
@@ -0,0 +1,39 @@
+package com.Deny.MyMentorProject.Controllers;
+
+import com.Deny.MyMentorProject.Services.UserService;
+import com.Deny.MyMentorProject.User.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserMainController {
+
+    @Autowired
+UserService userService;
+
+    public UserMainController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping
+   public String helloMessage() {
+
+        return userService. helloMessage();
+    }
+
+    @GetMapping("/greetUser")
+    public String helloUser() {
+
+        return "hello  " + userService.helloUser();
```

Comment:
The point of this feature was to be able to accept a name and greet that person. You should not hard-code Joe into your app. Have it work for any name the consumer passes to you.